### PR TITLE
Add GH sponsors webhook

### PR DESCRIPTION
**Pull request by iHiD**

### CODE REVIEW

File: app/jobs/process_github_sponsor_update_job.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class ProcessGithubSponsorUpdateJob < ApplicationJob
```
Comment (ErikSchierboom) [inline app/jobs/process_github_sponsor_update_job.rb:1]: Could this be a regular command with the special Mandate sidekiq magic?

Comment (iHiD) [reply app/jobs/process_github_sponsor_update_job.rb:1]: Sure!

**Comment by ErikSchierboom** [comment]
@iHiD Was there anything else we need to do for this PR?

**Comment by iHiD** [comment]
No, this is great, thanks. Can you merge and run the batch updater when you're ready pls :)

**Review by ErikSchierboom — APPROVED** [review]

**Comment by ErikSchierboom** [comment]
> Can you merge and run the batch updater when you're ready pls :)

Ran the batch updater

**Pull request merged**